versions.tf

```
@@ -2,7 +2,7 @@ terraform {
   required_version = ">= 0.12.0, < 0.14.0"
```

```suggestion
  required_version = ">= 0.12.0"
```

please also remove the upper limit for TF version, we don't use upper limit anymore

---

Sure, done in bd4d0fe